shared/src/packages/articles/libs/enums/articles-api-path.enum.ts

```diff
@@ -3,6 +3,9 @@ const ArticlesApiPath = {
   $ID: '/:id',
   REACT: '/react',
   OWN: '/own',
+  SHARE: '/:id/share',
+  SHARED_BASE: '/shared',
+  SHARED: '/shared/:token',
```

Comment:
Make also `SHARED_$TOKEN = '/shared/:token'` in constants and use it in place where you need to define route for page on frontend and to declare a sharing link on backend
```suggestion
  $ID_SHARE: '/:id/share',
  SHARED_$TOKEN: SHARED_$TOKEN,
```